package/Runtime/Scripts/Internal/IOSAdManager.cs

```diff
@@ -0,0 +1,63 @@
+﻿using UnityEngine;
+
+namespace Nimbus.Runtime.Scripts.Internal
+{
+    internal class IOSAdManager : MonoBehaviour
+    {
+        private NimbusAdUnit adUnit;
+
+        private static IOSAdManager instance;
+
+        internal static IOSAdManager Instance
+        {
+            get
+            {
+                if (instance == null)
+                {
+                    var obj = new GameObject("IOSAdManager");
```

Comment:
Can you change this to NimbusIOSAdManager to ensure we don't conflict with other adtech

Reply:
Done